### PR TITLE
add test for Date

### PR DESCRIPTION
**Pull request by linclark**
This adds a simple test for the Date output.

### CODE REVIEW

**Comment by bgrins** [comment]
How do I run these locally?  I'm running: `./node_modules/mocha/bin/mocha /Users/bgrinstead/Sites/console-output-frontend/js/components/Message/Types/JSObject/__tests__/`

And get:

    /console-output-frontend/js/components/Message/Types/JSObject/__tests__/Date.test.js:1
    (function (exports, require, module, __filename, __dirname) { import React from 'react';
                                                              ^^^^^^
    SyntaxError: Unexpected reserved word

**Comment by bgrins** [comment]
Ah, disregard that.. found `npm test`

File: js/components/Message/Types/JSObject/__tests__/Date.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```
Comment (bgrins) [inline js/components/Message/Types/JSObject/__tests__/Date.test.js]: As we discussed, let's move all of the tests into a single folder somewhere so it's easier to port into tree (maybe in the root level test directory)

Comment (linclark) [inline js/components/Message/Types/JSObject/__tests__/Date.test.js]: Updated.

**Comment by linclark** [comment]
Whoops, accidentally merged this with a fix to the webpack config (to make it possible to use dependencies that are npm linked). LMK if you had any other issues with it and I'll fix them.

**Pull request merged**